app/src/main/res/layout/list_new_bp_button.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  android:layout_width="match_parent"
+  android:layout_height="wrap_content">
+
+  <androidx.appcompat.widget.AppCompatButton
```

Comment:
As far as I know, we don't explicitly need to use `AppCompatButton` right? If we use `Button`, the framework automatically takes care of using `AppCompatButton` internally.

Reply:
I will change it to `Button`, I have gotten used to using `AppCompatButton` or `MaterialButton`